test/scenarios/releases/el96-lrel@fips.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+check_platform() {
+    if [[ "${UNAME_M}" =~ aarch64 ]] ; then
+        record_junit "setup" "scenario_create_vms" "SKIPPED"
+        exit 0
+    fi
+}
+
+scenario_create_vms() {
+    check_platform
+
+    prepare_kickstart host1 kickstart.ks.template "rhel-9.6-microshift-brew-optionals-4.${MINOR_VERSION}-${LATEST_RELEASE_TYPE}-fips"
```

Comment:
Can you please [add this check](https://github.com/openshift/microshift/blob/main/test/scenarios/releases/el96-lrel%40backups.sh#L8-L11) in case the image does not exist?